src/Native/LDANative/alias_multinomial_rng_int.cpp

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "alias_multinomial_rng_int.hpp"
+#include "rand_int_rng.h"
+#include <ctime>
+#include <list>
+#include <algorithm>
+#include <iostream>
+
+namespace wood
+{
+	AliasMultinomialRNGInt::AliasMultinomialRNGInt()
```

Comment:
>	 [](start = 0, length = 1)

I know what you just porting files from one repo to another, but....
Can we unify tab vs space story?
some files have tabs, some spaces